homeassistant/components/network/__init__.py

```diff
@@ -27,6 +29,19 @@
 CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
 
 
+def _not_using_host_networking() -> bool:
```

Comment:
the name should not have `not`. You are returning True if it is using host networking. 

Since the method checks both Docker and host network, maybe name it `_check_docker_without_host_networking` ?